acceptance/bundle/deploy/secret-scope/script

```diff
@@ -0,0 +1,15 @@
+envsubst < databricks.yml.tmpl > databricks.yml
+
+cleanup() {
+    trace $CLI bundle destroy --auto-approve
+}
+trap cleanup EXIT
+
+trace $CLI bundle deploy
+trace $CLI bundle summary --output json | jq '.resources.secret_scopes.secret_scope1'
+
+trace $CLI secrets list-scopes -o json | jq '.[] | select(.name == "my-secrets")'
```

Comment:
This is prone to be flaky for real tests; probability that a workspace has an existing "my-secrets" scope is real.

Reply:
good catch. added a random suffix to the name in a follow-up commit